cocodump/label_generator.py

```diff
@@ -0,0 +1,9 @@
+label_counter = 0
+
+
+def get_label() -> str:
+    global label_counter
+
+    label = f"L{label_counter}"
+    label_counter += 1
+    return label
```

Comment:
Module-shaped closure, heh.

Reply:
At least it does its job. However, it's a good idea to add underscore to `label_counter` so it will be "private".